doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+====
+Reef
+====
+
+Reef is the 18th stable release of Ceph. It is named after the reef squid
+(Sepioteuthis).
+
+v18.2.0 Reef
+============
+
+This is the first stable release of Ceph Reef.
+
+CephFS
+~~~~~~
+
+* CephFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* CephFS: It is now possible to delete the recovered files in the
+  ``lost+found`` directory after a CephFS post has been recovered in accordance
+  with disaster recovery procedures.
+
+MGR
+~~~
+
+* mgr/snap_schedule: The snap-schedule mgr module now retains one snapshot less
+  than the number mentioned against the config option 
```

Comment:
A space at the end of the line after `option` got inserted.

Reply:
Wild.

Fixed.